background/handle-settings-page.js

```diff
@@ -24,6 +24,13 @@ chrome.runtime.onMessage.addListener(function (request, sender, sendResponse) {
     chrome.storage.sync.set({
       addonsEnabled: scratchAddons.localState.addonsEnabled,
     });
+    chrome.tabs.query({}, (tabs) => {
+      // Tabs with urls means that ScratchAddons has perms there
+      tabs = tabs.filter((tab) => tab.url);
```

Comment:
Keep in mind this is not the case in Firefox - not even Scratch URLs have a url property. For consistency I should check how I did this in other parts of background code and do it the same way, not sure

Reply:
We use this for the settings page I believe...

Reply:
TODO: check